docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: "3"
+version: '3'
```

Comment:
is there a reason for the change from double to single quotes?

Reply:
Yeah, single quotes are in the prettier config file!